tests/TestCase.php

```diff
@@ -50,6 +48,8 @@ protected function setUp(): void
             ConsoleOutput::class,
             fn () => new TestConsoleOutput(),
         );
+
+        $this->http = $this->container->get(HttpRouterTester::class);
```

Comment:
I temporarily added this one, so that I could already rewrite some tests to use the new syntax.